src/chart/SunburstChart.tsx

```diff
@@ -218,10 +211,9 @@ export const SunburstChart = ({
           role="region"
         >
           <Surface width={width} height={height}>
-            {children}
             <Layer className={layerClass}>{sectors}</Layer>
+            {children}
           </Surface>
-          {renderTooltip()}
```

Comment:
Sunburst now renders tooltip only once so I could enable tests for it!

I will fix the other charts too if we find this approach acceptable.

Tooltip still renders everywhere even on this branch.